### DISCUSSION TITLE
Overflow-x

### DISCUSSION THREAD

**Discussion by anhthu253** [neuefische/questions#317] [Web]
Hi i was trying to set overflow-x to a section but it always came out overflow-y. The section includes several text span.
Anybody knows why?

**Comment by F-Kirchhoff** [comment]
Can you please send a link to a working deployment of your code, as well a link to the source code and the relevant file / code line? Thanks!

**Reply by anhthu253** [reply]
u meant a link to this problem? Phil solved it already 😃
